from networksecurity.exception.exception import NetworkSecurityException
import sys
from networksecurity.logging import logger


if __name__=='__main__':
    try:
        logger.logging.info("Enter the try block")
        a=1/0
        print("This will not be printed",a)
    except Exception as e:
           raise NetworkSecurityException(e,sys)